tools/acorn-optimizer.js

```diff
@@ -764,6 +764,103 @@ function applyDCEGraphRemovals(ast) {
   });
 }
 
+function createNode(props) {
+  var fakeParser = {
```

Comment:
is this object modified by `Node()`? if not, could it be in the global scope, to avoid allocation here?

Reply:
It's not modified no, so could allocate once. Just need something with an empty 'options' object property on it to let the constructor pass. :) Will adjust that.

Reply:
Actually it was easy to just allocate a real Parser for safety, and it works the same. Just did that and kept it. :)